pkg/service/backup/purger_test.go

```diff
@@ -51,7 +51,7 @@ func TestStaleTags(t *testing.T) {
 	x.Temporary = true
 	manifests = append(manifests, x)
 
-	tags := staleTags(manifests, RetentionMap{task0: {0, 3}, task1: {0, 2}})
+	tags, _ := staleTags(manifests, RetentionMap{task0: {0, 3}, task1: {0, 2}})
```

Comment:
You have a great opportunity to test here if `staleTags` returns the oldest tag the correct way. Please don't ignore the `oldest` value from `staleTags` return but include it into the assertion.

Reply:
I also extended `staleTags` for mixed retention policies.